data/sql/updates/pending_db_world/rev_1623407321415010500.sql

```diff
@@ -0,0 +1,27 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1623407321415010500');
+
+-- Add correct gossip text
+DELETE FROM `gossip_menu` WHERE `MenuID` IN (2991, 2992, 2993);
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES
+(2991, 3674),
+(2992, 3675),
+(2993, 3676);
+
+-- Add chat emote text
+DELETE FROM `creature_text` WHERE `CreatureID` = 10637 AND `GroupID` = 1;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`,`Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(10637, 1, 0, 'Malyfous turns and points to his catalogue on the barrel.', 16, 0, 100, 25, 0, 0, 0, 0, 'Malyfous Darkhammer');
+
+UPDATE `creature_text` SET `Emote` = 0 WHERE `CreatureID` = 10637 AND `GroupID` = 0;
+
+-- Add script on gossip end
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 10637;
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` IN (10637, 1063700));
```

Comment:
@Raelorasz please include the full `PRIMARY KEY` in the `DELETE`, in this case also the `source_type` is needed, so we don't risk deleting extra rows unintentionally.